settings.yml

```
@@ -17,3 +17,11 @@ subdomain_tracking:
   type: bool
   default: false
   description: Track visitors across main domain & subdomains, assuming discourse is on a subdomain.
+do_not_track:
+  type: bool
+  default: true
```

```suggestion
  default: false
```

Having this as `default: true` will force `do_not_track` to all existing consumers of the theme component. Can we start with default false here?